### PR TITLE
Move frontend build into the console image

### PR DESCRIPTION
**Pull request by coledave**
To facilitate building the static docker images move the frontend build tasks into the console image.

### CODE REVIEW

File: docker/image/console/root/lib/task/build/backend.sh.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    
+    if [ ! {{ @('backend.build.when')|raw }} ]; then
+        return 0;
+    fi
+
+    {% for step in @('backend.build.steps') -%}
+    {{ step|raw }}
+    {% endfor %}
+
+    set +e
```
Comment (kierenevans) [inline docker/image/console/root/lib/task/build/backend.sh.twig]: No need to `set +e` when within a subshell `(...)`

File: docker/image/console/root/lib/task/build/frontend.sh.twig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    
+    if [ ! {{ @('frontend.build.when')|raw }} ]; then
+        return 0;
+    fi
+
+    {% for step in @('frontend.build.steps') -%}
+    {{ step|raw }}
+    {% endfor %}
+
+    set +e
```
Comment (kierenevans) [inline docker/image/console/root/lib/task/build/frontend.sh.twig]: No need to set +e when within a subshell `(...)`

**Review by kierenevans — APPROVED** [review]

File: docker/image/console/root/lib/task/build/backend.sh.twig (hunk 1 above)
Comment (coledave) [reply docker/image/console/root/lib/task/build/backend.sh.twig]: updated

File: docker/image/console/root/lib/task/build/frontend.sh.twig (hunk 2 above)
Comment (coledave) [reply docker/image/console/root/lib/task/build/frontend.sh.twig]: updated

**Pull request merged**